stdlib/public/core/StringIndexConversions.swift

```diff
@@ -48,10 +48,10 @@ extension String.Index {
     _ unicodeScalarIndex: String.UnicodeScalarIndex,
     within other: String
```

Comment:
It is good that we added the String argument here even though we didn't need it before!